paper_io_sim.tex

```diff
@@ -759,7 +763,10 @@
         when application tasks release anonymous memory. Simulated cached
         memory is highly consistent with real values, except toward the end
         of Read 3 where it slightly increases in simulation but not in
-        reality \tristan{why?}. As expected, dirty data remains under the
+        reality. Our explanation could be that we allocate anonymous memory 
+        chunk-by-chunk to application, while anonymous memory is allocated 
+        when as soon as task starts.
+        As expected, dirty data remains under the
```

Comment:
I don't find this explanation very convincing: if it was true then we would see a step in the real memory consumption when the task would start, am I wrong?

Reply:
Yes you're right. Another explanation could be:
"In reality, because File 3 is entirely in cache, the cache amount does not increase when File 3 is read. While in simulation, a part of File 3 is read from disk, leading to an increase in cache amount followed by a decrease when we call eviction."